manual/guides/writing/checklist.md

```diff
@@ -73,20 +73,28 @@ remember some people reading it won't know as much jargon as you do.
 ```markdown
 - [ ] "Data" is plural (the data are critical).
 - [ ] Compare to (point out similarities between different things) vs.  compared with (point out differences between similar things)
+- [ ] Elemental symbols (Ni, Li, Na, Pu) are capitalized, but their names are not (nickel, lithium, sodium, plutonium).
+- [ ] Do not use the word "where" unless referring to a location (try "such that" or "in which").
+- [ ] Avoid run-on sentences.
+- [ ] The preposition "of" shows belonging, relations, or references. The preposition "for" shows purpose, destination, amount, or recipients. They are not interchangeable.
+```
+
+## Enhancing Punctuation
+
+```markdown
 - [ ] Commas and periods go inside end quotes, except for when there's a parenthetical reference afterwards.
 - [ ] Colons and semicolons go outside closed quotations.
 - [ ] Semicolon: connects two independent clauses. OR separates items when the list contain internal punctuation.
 - [ ] Use a colon to introduce a list, quote, explanation, conclusion, or amplification.
-- [ ] Elemental symbols (Ni, Li, Na, Pu) are capitalized, but their names are not (nickel, lithium, sodium, plutonium).
 - [ ] The Oxford comma must appear in lists ("lions, tigers, and bears").
-- [ ] Do not use the word "where" unless referring to a location (try "such that" or "in which").
-- [ ] Avoid run-on sentences.
-- [ ] The preposition "of" shows belonging, relations, or references. The preposition "for" shows purpose, destination, amount, or recipients. They are not interchangeable.
+- [ ] Use hyphens to join words acting as a single adjective before a noun (e.g., "well-known prankster"), not after a nount (e.g., "the prankster is well known").
+- [ ] Two words joined by a hyphen in title-case should both be capitalized.
+- [ ] Hyphens join a prefix and a capitalized word, figures, or letters; compound numbers; words to the prefixes ex, self, and all; and words to the suffix elect.
```

Comment:
I don't really understand the meaning of this sentence, could you make it less dense? And give some examples?

Reply:
Oh totally! Yeah, it is a bit dense